httpclient5-cache/src/main/java/org/apache/hc/client5/http/cache/HttpCacheStorage.java

```diff
@@ -94,4 +94,14 @@ void updateEntry(
      */
     Map<String, HttpCacheEntry> getEntries(Collection<String> keys) throws ResourceIOException;
 
+    /**
+     * Check if the cache is still active and has not shut down.
+     *
+     * @return {@code true} if the cache is active, otherwise return {@code false}.
+     * @since 5.2
+     */
+    default boolean isActive() {
```

Comment:
@arturobernalg Why adding this method to `HttpCacheStorage` when it clearly applies to `ManagedHttpCacheStorage` only and it makes no sense for all other implementations?

Reply:
> ManagedHttpCacheStorage

that make sense. Didn't see the other implementations. My bad.
Changed only for `ManagedHttpCacheStorage `